docs/fidesops/docs/saas_connectors/example_configs/hubspot.md

```diff
@@ -0,0 +1,215 @@
+
+# Hubspot
+
+## Implementation Summary
```

Comment:
I really like the way you linked out to the API docs and summarized the available actions in a single place. What are your thoughts on adding something like a `Right to Update` column? Each endpoint can have a `read`, `update`, or `delete` so it's important to make the distinction between update and delete.

Reply:
Is it the case that we use "Update" to satisfy a "Delete" - ie, updating information where it cannot be deleted? If that's the case I think I'd rather change "Right to Delete" to be inclusive of Update endpoints, otherwise just make this a simple `read`/`update`/`delete` breakdown (without promising it satisfies right to access or right to delete).